### DISCUSSION TITLE
Haml-lint integration

### DISCUSSION THREAD

**Discussion by frothedoatmilk** [zed-industries/zed#19917] [Feature Requests]
I work a lot with haml, and I'm appreciative for the [zed-haml](https://github.com/davidcornu/zed-haml) extension but I can't figure out how to integrate [haml-lint](https://github.com/sds/haml-lint). I've read in the docs that linting is usually performed by an LSP. But haml-lint is not an LSP, it's just a plain old linter. Any advice?

**Comment by bajrangCoder** [comment]
You will need to wrap haml linter as LSP then it will work with zed. Check online for creating LSP.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
